### ISSUE TITLE
Improvment Suggestion for Relationships Documentation

### ISSUE THREAD

**Issue by JHerculesVW** [keystonejs/keystone#4791]
Having just been struggling with getting [Relationship Definitions to work](https://keystonejs.netlify.com/documentation/database/relationships/), the best thread that actually made me understand how to use this function came from one Steve Holgado [on this Stack Overflow post](https://stackoverflow.com/questions/46258439/keystonejs-display-many-to-many-relationships-in-the-model-adminui). He says:

>Regarding the relationship options:
>**path** is the name you want for the field on the model.
>**ref** is the referred Model (the one that has the relationship field).
>**refPath** is the field in the referenced model that has the relationship.

Furthermore, the example code he provided showed something that I didn't pick up from the official documentation: that the **refPath** in .relationship needs to be the same name as the declared Types.Relationship field in the referring model.

In short: the docs could be more clear that these two items need to use the same field name. Basically, if I had to write it, I would tweak Steve's explanation to say:

>**path** is the field name you want to use in the current Model.
>**ref** is the key or name of the referred Model (one which has a Types.Relationship field that points at the current Model).
>**refPath** is the name of the Types.Relationship field in the referred model.

This way, the nomenclature is comparable between explanations of each field and where each belongs, hopefully making it easier to follow.
